internal/config/config_test.go

```diff
@@ -274,3 +274,51 @@ func TestLoadConfig_ValidBooleanValues(t *testing.T) {
 		t.Errorf("LoadConfig() ReadOnly = %v, want true", cfg.ReadOnly)
 	}
 }
+
+func TestLoadConfig_ValidIntValue(t *testing.T) {
+	// Set required env variables for basic validation to pass
+	t.Setenv("NEO4J_URI", "bolt://localhost:7687")
+	t.Setenv("NEO4J_USERNAME", "testuser")
+	t.Setenv("NEO4J_PASSWORD", "testpass")
+
+	t.Run("default value", func(t *testing.T) {
+		// Unset the env var to test default
+		t.Setenv("NEO4J_SCHEMA_SAMPLE_SIZE", "")
+
+		cfg, err := LoadConfig(nil)
+		if err != nil {
+			t.Fatalf("LoadConfig() unexpected error: %v", err)
+		}
+
+		if cfg.SchemaSampleSize != 100 {
+			t.Errorf("LoadConfig() SchemaSampleSize = %v, want 100", cfg.SchemaSampleSize)
```

Comment:
Just a thought, not really needing anything here, but we have magic numbers here for default values